CODE_OF_CONDUCT.md

```diff
@@ -3,32 +3,17 @@
 We value the participation of every member of our community and want to ensure that every contributor has an enjoyable and fulfilling experience.
 Accordingly, everyone who participates in the Turing Way project is expected to show respect and courtesy to other community members at all times.
 
-Kirstie Whitaker, as PI of this project, and all project members, are dedicated to a ***harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
+*The Turing Way* Core Staff and our volunteer leaders, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

Comment:
I think there's something important about naming the leaders. Can we add me and @malvikasharan back in?

Also - is there a definition of core staff we can link to yet?

If not my preference would be to keep this as named leaders (K & M) but to incorporate this edit when we have more of the governance developed?

Reply:
```suggestion
*The Turing Way* project leaders, Kirstie Whitaker and Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

Reply:
Added the names back in. Agree on linking to the governance documents when they are live in the book